java/test/android/controller/src/main/java/com/zeroc/testcontroller/ControllerApp.java

```diff
@@ -35,11 +32,11 @@ private static class TestSuiteBundle {
             _class = (Class<? extends test.TestHelper>) _loader.loadClass(name);
         }
 
-        test.TestHelper newInstance() throws IllegalAccessException, InstantiationException {
+        test.TestHelper newInstance() throws ReflectiveOperationException {
             if (_class == null) {
                 return null;
             }
-            return _class.newInstance();
```

Comment:
`Class::newInstance` has been deprecated for years:
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Class.html#newInstance()
`Class::getDeclaredConstructor` is the proper way to do this in Java.